scripts/UnioneApiChecker.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace UnioneScripts;
+
+use Composer\IO\IOInterface;
+use Composer\Script\Event;
+use Unione\UnioneClient;
+
+class UnioneApiChecker
+{
+    /**
+     * The UnioneClient client.
+     *
+     * @var UnioneClient
+     */
+    private UnioneClient $client;
+
+    /**
+     * The script config file.
+     *
+     * @var array
+     */
+    private array $config;
+
+    /**
+     * The array with error messages.
+     *
+     * @var array
+     */
+    private array $messages = [];
+
+    public function __construct(array $args, array $config)
+    {
+        $this->client = new UnioneClient($args[1], $args[0]);
+        $this->config = $config;
+    }
+
+        /**
+         * Check Unione API methods.
+         *
+         * @param \Composer\Script\Event $event
+         *
+         * @return void
+         */
+        public static function testApi(Event $event)
+        {
+            $args = $event->getArguments();
+            $io = $event->getIO();
+
+            if (\count($args) < 2) {
+                $io->write('Please enter HOSTNAME and APIKEY parameters. Example: composer test UNIONE-HOSTNAME UNIONE-API-KEY'.PHP_EOL);
```

Comment:
Can you try `writeln()` method? Based on the documentation, it'll add a new line at the end, so you won't need to add `PHP_EOL` at the end.